test/parallel/test-os-eol.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const os = require('os');
+
+const eol = common.isWindows ? '\r\n' : '\n';
+
+assert.strictEqual(eol, os.EOL);
+
+common.expectsError(function() {
```

Comment:
IMHO I'd like to use anonymous function.

Refs: https://github.com/nodejs/node/issues/14496#issuecomment-318011509

Even though there was no result about that discussion.